HoloPipelines/pipelineServer.py

```diff
@@ -0,0 +1,202 @@
+from flask import Flask, request
+from pipelineController import startPipeline, getPipelineList
+from flask_json import json_response
+from datetime import datetime
+from pathlib import Path
+import uuid
+import json
+import threading
+import time
+import logging
+import requests
+import os
+import pathlib
+from zipfile import ZipFile
+
+app = Flask(__name__)
+app.config["JSON_ADD_STATUS"] = False
+
+# global variables
+this_cwd = pathlib.Path.cwd()
+save_directory = Path("input")
+output_directory = Path("output")
+pipeline_list = getPipelineList()
+status = {
+    "j0": {"status": "segment", "stamp": "2019-08-05 14:09:19"},
+    "j1": {"status": "segment", "stamp": datetime.now().strftime("%Y-%m-%d %H:%M:%S")},
+}
+jobid2plid = {"j0": "nifti2glb", "j1": "dicom2glb"}
```

Comment:
None of these three dictionaries are needed. job to plid, job to inputURL, job to outputURL. Please remove all.